bower.json

```diff
@@ -1,30 +1,53 @@
 {
   "name": "lazy-imports",
+  "version": "2.0.0",
```

Comment:
Great, I'm good with 2.0.0 if you are, just wasn't sure if there was some strict semver that would allow us to bump 1 major but semver is really more guidelines. I think it would be better to keep is 2.0.0 for consistency.